.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
Can we keep this as 18 for the next 8 days 😉

Reply:
won't get merged in the next 8 days :)